services/date.js

```diff
@@ -64,3 +63,7 @@ module.exports.getZodiacSign = (id) => {
       });
   });
 };
+
+module.exports.dateToZodiac = (date) =>
```

Comment:
did you see my previous comment about this function? I think it did not show up. I tested this and it is a bit wonky, I think we need to add a unit test for this function